pkg/reconciler/brokercell/brokercell_test.go

```diff
@@ -140,12 +150,62 @@ func TestAllCases(t *testing.T) {
 			},
 			WantErr: true,
 		},
+		{
+			Name: "Ingress HorizontalPodAutoscaler.Create error",
```

Comment:
Looks like we can add a couple of new test cases:
 - HPA exists with same spec and no updates needed
- HPA exists with different spec and update succeeds